java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S3331_java.html

```diff
@@ -0,0 +1,25 @@
+<p>A cookie's domain specifies which websites should be able to read it. Left blank, browsers are supposed to only send the cookie to sites that
```

Comment:
Not directly related to implementation, but I think it's a shame that `HttpCookie` is mentioned nowhere in RSPEC description, not even as an example

Reply:
I'll update the example in RSPEC.